GVFS/GVFS.Common/GVFSConstants.cs

```diff
@@ -210,7 +210,9 @@ public static class VerbParameters
         {
             public static class Mount
             {
-                public const string ServiceName = "internal_use_only_service_name";
+                public const string InternalParameters = "internal_use_only_parameters";

```

Comment:
Not your doing, but this parameter doesn't belong in `VerbParameters.Mount`, since it's not specific to just `MountVerb` (I believe it was when it was first added, and later got moved to all verbs).

I would create a class named `VerbParameters.Internal` with a member named `Name` and other members for each of the parameters that it can contain.

Reply:
Or even better, define a class with properties, and use Json to serialize/deserialize its members. There's no need for string-typed values here since we don't need any cross-version compatibility.

Reply:
I updated it to be a serialized Json class.  It's a bit more painful (supplying json across a command line is no fun!), but that will probably encourage folks not to use it which is what we want, while forcing type.